miniscope_io/stream_daq.py

```diff
@@ -177,7 +178,8 @@ def _trim(
         expected_payload_size = expected_size_array[0]
         expected_data_size = expected_size_array[header.frame_buffer_count]
 
-        if data.shape[0] != expected_payload_size:
+        # This is temporary. It's better to detect all dummy words and remove them.
```

Comment:
This refers to (next line):
```
if data.shape[0] != expected_payload_size + self.config.dummy_words * 4:
```
I left this comment because it feels cleaner to remove these dummy words right after the preamble detections. That way, all data we inject into later stages will be pure metadata and pixel data. I'll put this in an inline to do.

Reply:
So, it's more a flag for refactoring.

Reply:
Ok great! Thats what I figured